src/zope/security/checker.py

```diff
@@ -24,10 +24,10 @@
 (:func:`getCheckerForInstancesOf`, :func:`selectChecker`).
 
 This module is accelerated with a C implementation on CPython by
-default. If the environment variable ``PURE_PYTHON`` is set (to any
-value) before this module is imported, the C extensions will be
-bypassed and the reference Python implementations will be used. This
-can be helpful for debugging and tracing.
+default. If the environment variable ``PURE_PYTHON`` is set (to something else
```

Comment:
You now expect the value to be an integer.

I suggest to allow (for all affected packages): interpret a missing, empty an "0" value for the envvar `PURE_PYTHON` as "not PURE_PYTHON". The logic could look something like `PURE_PYTHON = int(os.environ.get("PURE_PYTHON") or 0)`

Reply:
I pushed a new change to change the wording to make it more clear that an int is expected.

> If the environment variable ``PURE_PYTHON`` is set to ``1`` before this module is imported

In my understanding, all affected packages use a logic similar to `int(os.environ.get("PURE_PYTHON") or 0`